### ISSUE TITLE
[Feature Request] Add ability to quickly add a new site

### ISSUE THREAD

**Issue by Whoaa512** [borismus/keysocket#155]
Add a quick add feature where you click a button to enter in class names and a regex to match the site url.

**Comment by zwacky** [comment]
Really like this one.

This would be the start of a tabbed popup, Along with #82.
- a **tab control** tab for #82
- a **add/remove/edit custom config** tab for this issue

I'd suggest default bootstrap as the _design guidelines_.

**Comment by feedbee** [comment]
As I can see, this should be Options windows. By clicking icon extension should do:
1. If current tab is not registered (not active), register this tab and unregister all others.
2. If current tab is registered (active), unregister itself.

All permanent configuration (like manage services, setup tab control) should be done in options.

**Comment by zwacky** [comment]
makes sense to have the 'what tab gets the control' action as prominent as possible - therefore the icon click itself. the config as an option seems fine, since it's not something a lot of users will do.
